yal/servicefinder.py

```diff
@@ -251,8 +247,8 @@ async def next_(choice: Choice):
         question = self._(
             "\n".join(
                 [
-                    "🏥 Find Clinics and Services",
-                    metadata["servicefinder_breadcrumb"],
+                    "NEED HELP? / Find clinics and services / "
+                    f"{metadata['servicefinder_breadcrumb']}",
```

Comment:
Does this need to be an fstring if its only the one value?

Reply:
@erikh360  Yeah it needs to be on a separate line because the line is too long but in the message it has to be on the same line